browser.js

```diff
@@ -1,59 +1,68 @@
 /* eslint-env browser */
 'use strict';
 import pEvent from 'p-event';
+import isIp from 'is-ip';
 
-const ipv4 = async ({isSecondTry = false} = {}) => {
-	const peerConnection = new RTCPeerConnection({iceServers: []});
+const getIp = async ({isSecondTry = false} = {}) => {
+	try {
+		const peerConnection = new RTCPeerConnection({iceServers: []});
 
-	peerConnection.createDataChannel('');
-	peerConnection.createOffer(peerConnection.setLocalDescription.bind(peerConnection), () => {});
+		peerConnection.createDataChannel('');
+		peerConnection.createOffer(peerConnection.setLocalDescription.bind(peerConnection), () => { });
 
-	const {candidate} = await pEvent(peerConnection, 'icecandidate', {
-		timeout: 10000
-	});
+		const {candidate} = await pEvent(peerConnection, 'icecandidate', {
+			timeout: 10000
+		});
 
-	peerConnection.close();
+		peerConnection.close();
 
-	if (candidate && candidate.candidate) {
-		const result = candidate.candidate.split(' ')[4];
-		if (result.endsWith('.local')) {
-			if (isSecondTry) {
-				return;
-			}
+		if (candidate && candidate.candidate) {
+			const result = candidate.candidate.split(' ')[4];
+			if (result.endsWith('.local')) {
+				if (isSecondTry) {
+					return;
+				}
 
-			const inputDevices = await navigator.mediaDevices.enumerateDevices();
-			const inputDeviceTypes = new Set(inputDevices.map(({kind}) => kind));
+				const inputDevices = await navigator.mediaDevices.enumerateDevices();
+				const inputDeviceTypes = new Set(inputDevices.map(({kind}) => kind));
 
-			const constraints = {};
+				const constraints = {};
 
-			if (inputDeviceTypes.has('audioinput')) {
-				constraints.audio = true;
-			} else if (inputDeviceTypes.has('videoinput')) {
-				constraints.video = true;
-			} else {
-				return;
-			}
+				if (inputDeviceTypes.has('audioinput')) {
+					constraints.audio = true;
+				} else if (inputDeviceTypes.has('videoinput')) {
+					constraints.video = true;
+				} else {
+					return;
+				}
+
+				const mediaStream = await navigator.mediaDevices.getUserMedia(constraints);
+				for (const track of mediaStream.getTracks()) {
+					track.stop();
+				}
 
-			const mediaStream = await navigator.mediaDevices.getUserMedia(constraints);
-			for (const track of mediaStream.getTracks()) {
-				track.stop();
+				return await getIp({isSecondTry: true});
 			}
 
-			return ipv4({isSecondTry: true});
+			return result;
 		}
-
-		return result;
-	}
+	} catch { }
```

Comment:
```suggestion
	} catch {}
```

Other places too. Please disable Prettier and any other auto-formatters when you work upstream repos.